cmd/exporter/Dockerfile

```diff
@@ -1,15 +1,27 @@
-FROM golang:1.13.5 as builder
+FROM golang:1.17 as builder
 WORKDIR /workspace
-COPY . .
+
 ENV GOPROXY=https://goproxy.cn,direct
+
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY cmd/ cmd/
+COPY pkg/ pkg/
+
+# Build
 RUN CGO_ENABLED=0 GO111MODULE=on go build -a -o ./bin/kube-events-exporter ./cmd/exporter/main.go
 
-FROM alpine:3.9
+# Use distroless as minimal base image to package the manager binary
+# Refer to https://github.com/GoogleContainerTools/distroless for more details
+FROM kubespheredev/distroless-static:nonroot
```

Comment:
`kubesphere/distroless-static` is the latest, not `kubespheredev/distroless-static:nonroot`